packages/datasources/javascript/index.js

```diff
@@ -0,0 +1,44 @@
+import { EvidenceType } from '@evidence-dev/db-commons';
+import { pathToFileURL } from 'url';
+
+export const options = {};
+
+/**
+ * @type {import("@evidence-dev/db-commons").GetRunner<ConnectorOptions>}
+ */
+export const getRunner = () => {
+	return async (queryText, queryPath) => {
+		if (queryPath.endsWith('.js')) {
+			const module = await import(pathToFileURL(queryPath).href);
+			const data = module.data;
+
+			if (module.data) {
+				const columnNames = Object.keys(data[0]);
+
+				const output = {
+					rows: data,
+					columnTypes: columnNames.map((name) => ({
+						name,
+						evidenceType:
+							typeof data[0][name] === 'number'
+								? EvidenceType.NUMBER
+								: data[0][name] instanceof Date
+									? EvidenceType.DATE
+									: EvidenceType.STRING,
```

Comment:
This doesn't seem to have support for booleans

Reply:
We also have this logic built-in to `db-commons`:
https://github.com/evidence-dev/evidence/blob/next/packages/lib/db-commons/index.cjs#L31-L88